patches/eslint+7.23.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/eslint/lib/linter/linter.js b/node_modules/eslint/lib/linter/linter.js
+index adb5c21..4a4be92 100644
+--- a/node_modules/eslint/lib/linter/linter.js
++++ b/node_modules/eslint/lib/linter/linter.js
+@@ -560,7 +560,7 @@ function resolveParserOptions(parserName, providedOptions, enabledEnvironments)
+  */
+ function resolveGlobals(providedGlobals, enabledEnvironments) {
+     return Object.assign(
+-        {},
++        Object.create(null),
```

Comment:
For some reason, the properties being assigned include `Object.prototype` properties, and assigning them in this manner appears to walk the prototype chain. This doesn't work under LavaMoat, because the prototype is frozen. Creating an object with a `null` prototype seems to work just fine.